questions/questions/intervals.py

```diff
@@ -109,3 +109,42 @@ def _generate_question_type(self):
 
     def _generate_question_params(self):
         self._question_params = {}
+
+
+class InvertedInterval(Question):
+    interval_qualities = ['Major', 'Minor', 'Augmented', 'Diminished', 'Perfect']
```

Comment:
Let's use the list from _utilities so have DRY.

Reply:
very cool